functions.php

```diff
@@ -6,7 +6,7 @@ function scripts_and_styles_method() {
   // library.js is to bundle plugins. my.js is your scripts. enqueue more files as needed
   $jslib = $templateuri."library.js";
   wp_enqueue_script( 'jslib', $jslib,'','',true);
-  $myscripts = $templateuri."main.js";
+  $myscripts = $templateuri."main.min.js";
```

Comment:
I think this is a sourcemap configuration issue though. To do with the combo of uglify and sourcemap